api/routes/apds/get.js

```diff
@@ -72,6 +73,12 @@ module.exports = (
         logger.info(`id: ${req.params.id}, state: ${stateId}`);
         const apdFromDB = await getAPDByIDAndState(req.params.id, stateId);
 
+        const validation = await ldClient.variation(
```

Comment:
So we pass these params to LD: key name, user info stuff, and initial flag on/off (true/false)? That's nice and straightforward if I understand it correctly.